packages/cli/commands/logs.js

```diff
@@ -164,13 +92,74 @@ exports.handler = async options => {
   }
 };
 
+const appFunctionLog = async (accountId, options) => {
+  const { latest, follow, compact, functionName, appPath } = options;
+
+  let logsResp;
+
+  if (follow) {
+    const spinner = ora(
+      `Waiting for log entries for "${functionName}" on account "${accountId}".\n`
+    );
+    const tailCall = after =>
+      getAppFunctionLogs(accountId, functionName, appPath, { after });
+    const fetchLatest = () =>
+      getLatestAppFunctionLogs(accountId, functionName, appPath);
+
+    await tailLogs({
+      accountId,
+      compact,
+      spinner,
+      tailCall,
+      fetchLatest,
+    });
```

Comment:
One thing that I am not sure about is whether the indirection and passing in of functions is easy enough to follow.

Reply:
I think it's clear from the function name.